### PR TITLE
fix: content style issue in light mode

### PR DESCRIPTION
**Pull request by Gavin-Gong**
## PR Type

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context
- Some HTML tags(e.g. `h1`,`h2`) cant be seen in light mode.
- inline code and block code should probably be in a monospace font.

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality
- [X] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [X] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs)

#### Community participation

- [X] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

**before**
beta kodadot xyz_contribute (1)

**after**
localhost_9090_contribute

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
Hey! Thanks for spotting this one.
- Title color can be set using `$content-heading-color` [Bulma variable](https://bulma.io/documentation/elements/content/#variables) (check out `_derived-variables.scss` file)
- We'll try to avoid using `_light.scss` & `_dark.scss` (check out #4855)

otherwise code lgtm

**Comment by Gavin-Gong** [comment]
I think I have to wait for #4855  to be merged before I can continue my work.

**Review by roiLeo — COMMENTED** [review]
> I think I have to wait for #4855 to be merged before I can continue my work.

No need to wait for that, `ktheme` has already been implemented

you can add in `_typography.scss`
```scss
.content {
	h1,
	h2,
	h3,
	h4,
	h5,
	h6,
	pre:not(.hljs),
  	strong {
		@include ktheme() {
			color: theme('text-color');
		}
	}
}
```

File: styles/components/_content-markdown.scss [hunk 1]
```diff
@@ -1,10 +1,11 @@
 @import '~/node_modules/highlight.js/styles/github-dark.css';
+@import '@/styles/abstracts/variables';
 
 .content {
-  pre,
-  blockquote {
-    background-color: black;
-    border-left: 5px solid #dbdbdb;
-    padding: 1.25em 1.5em;
```
Comment (Gavin-Gong) [inline styles/components/_content-markdown.scss:8]: Bluma has already built in, so remove it.
```css
{
    border-left: 5px solid #dbdbdb;
    padding: 1.25em 1.5em;
}
```

**Review by roiLeo — APPROVED** [review]
✅ code lgtm & wfm

**Comment by vikiival** [comment]
Thank you for your contribution!

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 33.28 USD/KSM ~ 0.901 $KSM
🧗 [GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs](https://kusama.subscan.io/account/GqheyaQ3c8SFBJYorTXyLW6jvLXKNAXzbad77NXoLYJBXXs)
🔗 [0x090f1a068ffb6d1e2c7a5befb42f0ba7dce2a6e12038b8dfee70086178fabaf2](https://kusama.subscan.io/extrinsic/0x090f1a068ffb6d1e2c7a5befb42f0ba7dce2a6e12038b8dfee70086178fabaf2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**